packages/react-components/src/components/Input/Input.stories.tsx

```diff
@@ -49,3 +51,23 @@ export const States = (): JSX.Element => (
     </StoryDescriptor>
   </>
 );
+
+export const Kinds = (): JSX.Element => (
+  <>
+    <StoryDescriptor title="Text">
+      <Input placeholder={placeholderText} />
```

Comment:
While defining specific variants, we should not rely on `default` values as it's intended to include prop value for docs